web-server_route.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: web-server
+  namespace: k8sinitiative
+  resourceVersion: "72299960"
+  selfLink: /apis/route.openshift.io/v1/namespaces/k8sinitiative/routes/web-server
+  uid: 18489824-19e1-4965-8cbb-f2f3cf2d05b4
```

Comment:
I don't think these attributes are necessary, are them? how did you generate them?

Reply:
These 3 items: `resourceVersion`, `selfLink` and `uid` are  automatically added when you run `oc apply`
I was unsure if commit it or not, I couldn't find anything useful in the docs, but making a [quick search on GH](https://github.com/search?q=selfLink%3A+%2Fapis%2Froute.openshift.io%2Fv1%2F&type=code) I've found out that people is actually committing

Reply:
But, deleting them and creating the route again is working fine. 
I think we can delete them